R/http.R

```diff
@@ -157,6 +162,9 @@ TWIT_paginate_max_id <- function(token, api, params,
     if (length(id) == 0) {
       break
     }
+    if(i>length(results)) { #doubling size, based on https://github.com/ropensci/rtweet/pull/567#issuecomment-821169712
```

Comment:
I'd recommend commenting with a link to https://en.wikipedia.org/wiki/Dynamic_array#Geometric_expansion_and_amortized_cost or similar, which explains why this approach is needed.

Reply:
thanks for the suggestion. did this.